docs/Doxyfile

```diff
@@ -2121,7 +2121,7 @@ PERL_PATH              = /usr/bin/perl
 # powerful graphs.
 # The default value is: YES.
 
-CLASS_DIAGRAMS         = YES
+CLASS_DIAGRAMS         = NO
```

Comment:
Are these no longer needed/used?

Reply:
We don't have diagrams in the docs and these cause a couple of hundred build warnings.